packages/nimble-components/src/anchor/tests/anchor.spec.ts

```diff
@@ -80,6 +80,25 @@ describe('Anchor', () => {
                 expect(element.control!.getAttribute(name)).toBe('foo');
             });
         });
+
+        it('for attribute tabindex', async () => {
+            await connect();
+
+            element.setAttribute('tabindex', '-1');
+            await waitForUpdatesAsync();
+
+            expect(element.control!.getAttribute('tabindex')).toBe('-1');
+        });
+    });
+
+    it('should clear tabindex attribute from the internal control when cleared from the host', async () => {
+        await connect();
+
+        element.setAttribute('tabindex', '-1');
+        element.removeAttribute('tabindex');
```

Comment:
Intent is to test that going from `tabindex` being set to removing it also removes it from the inner anchor. That's why that `setAttribute('tabindex', '-1')` is there.

Reply:
I'd recommend putting in assertions between those transitions

Reply:
I noticed that I could move the `setAttribute` calls to before `connect()`, better indicating that it is a test precondition and separating it from the `removeAttribute` call. I generally try to avoid incidental assertions that are already covered by other tests (i.e. the `"should reflect value to the internal control for attribute tabindex"` test).